### ISSUE TITLE
The Settings page requires a password when updating any field

### ISSUE THREAD

**Issue by rschristian** [gothinkster/realworld#630]
Currently the "New Password" field is required by https://conduit.productionready.io/api when it should not be. The user is unable to say update their username on the Settings page without also providing a value in the "New Password" input, meaning they have to essentially keep overwriting their password in order to update any profile detail.

**Comment by geromegrignon** [comment]
It's pretty common to ask for authentication information when you are about to change sensitive information about the account.
It provides a way to secure those information if an user leaves thier computer unattended without locking their session.

As such changes are rare, it doesn't introduce much inconvenience for users

**Comment by rschristian** [comment]
Sure, that is pretty common, however, that is not what this is.

If you provide invalid credentials on that form your changes to the account aren't rejected. Instead, the credentials for the account change entirely. There is no confirmation of credentials built into the back end for that feature.

**Comment by rschristian** [comment]
If that is supposed to be a way in which the user needs to provide their credentials to change their bio, image, or username, and only those, there will probably also need to be a way to change their email and passwords.

**Comment by geromegrignon** [comment]
I agree. We can't affect all existing implementations with such changes but the goal is to work as soon as possible on new specs based on all suggestions provided to make them more 'real'
